mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala

```diff
@@ -359,38 +365,6 @@ private[ml] trait TreeEnsembleParams extends DecisionTreeParams {
       oldImpurity: OldImpurity): OldStrategy = {
     super.getOldStrategy(categoricalFeatures, numClasses, oldAlgo, oldImpurity, getSubsamplingRate)
   }
-}
-
-/**
- * Parameters for Random Forest algorithms.
- */
-private[ml] trait RandomForestParams extends TreeEnsembleParams {
```

Comment:
@MLnick 
 
Earlier featureSubsetStrategy, setFeatureSubsetStrategy, getFeatureSubsetStrategy were part of RandomForestParams , moved them to be part of TreeEnsembleParams , so that it can be accessed by both Random forest and GBT . I have not moved anything apart from that .